src/api/options-misc.md

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
I think we could get the title to be "Opções: Miscelânea", what do you guys think about that?

Reply:
@viniciusdeliz tu podes escrever e realizar comunicação em Português enquanto estivermos por aqui. :sweat_smile: 

@viniciusdeliz eu enviei-te um convite que tornar-te-á um colaborador deste projeto,
podes aceitar e começar a ajudar com a realização das fusões de mudanças, dê preferência
as fusões do tipo `squash and merge`.